app/helpers/content_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ContentHelper
+  def split_tag(value, delimiter)
+    if value.is_a? Array
+      return value unless value.join.include? delimiter
+      value.map { |str| str.partition(delimiter) }.flatten.reject(&:empty?)
```

Comment:
Ruby provides a faster way to map and then flatten via the `flat_map` method, i.e.:
```ruby
value.flat_map { |str| str.partition(delimiter) }.reject(&:empty?)
```